resources/leiningen/new/kraken_works/PROJECT@.service.template

```diff
@@ -24,7 +24,7 @@ ExecStartPre=-/usr/bin/docker rm ${CONTAINER}
 ExecStartPre=/bin/bash -c 'sleep 2 && curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/buildkite/dockercfg?raw -o /root/.dockercfg'
 ExecStartPre=/usr/bin/docker pull ${DOCKER_REPO}:${VERSION}
 
-ExecStart=/bin/bash -c 'docker run --name ${CONTAINER} --restart=always \
```

Comment:
Yes because systemd should be managing that, not Docker. There is already a `Restart` option you can put into the unit files themselves, and this was just making docker and systemd fight with each other about who should restart what, when. And letting Docker do it on its own meant that we could have containers running where fleet reports them as not running, which is always a good time.